### ISSUE TITLE
Configure Renovate - [merged]

### ISSUE THREAD

**Issue by iamjohnnym** [iamjohnnym/test-gl-mirror#6]
_Merges renovate/configure -> main_

Welcome to [Renovate](https://github.com/renovatebot/renovate)! This is an onboarding MR to help you understand and configure settings before regular Merge Requests begin.

🚦 To activate Renovate, merge this Merge Request. To disable Renovate, simply close this Merge Request unmerged.

---
### Detected Package Files

 * `.gitlab-ci.yml` (gitlabci)
 * `package.json` (npm)
 * `.nvmrc` (nvm)

### Configuration Summary

Based on the default config's presets, Renovate will:

  - Start dependency updates only once this onboarding MR is merged
  - Enable Renovate Dependency Dashboard creation.
  - If Renovate detects semantic commits, it will use semantic commit type `fix` for dependencies and `chore` for all others.
  - Ignore `node_modules`, `bower_components`, `vendor` and various test/tests directories.
  - Autodetect whether to pin dependencies or maintain ranges.
  - Rate limit MR creation to a maximum of two per hour.
  - Limit to maximum 10 open MRs at any time.
  - Group known monorepo packages together.
  - Use curated list of recommended non-monorepo package groupings.
  - A collection of workarounds for known problems with packages.
  - Rate limit MR creation to a maximum of one per hour.
  - Update lint packages automatically if tests pass.
  - Update testing packages automatically if tests pass.
  - Update `@types/*` packages automatically if tests pass.
  - Rebase existing MRs any time the base branch has been updated.

🔡 Would you like to change the way Renovate is upgrading your dependencies? Simply edit the `renovate.json` in this branch with your custom config and the list of Merge Requests in the "What to Expect" section below will be updated the next time Renovate runs.

---

### What to Expect

With your current configuration, Renovate will create 1 Merge Request:

<details>
<summary>chore(deps): pin dependencies</summary>

  - Schedule: ["at any time"]
  - Branch name: `renovate/pin-dependencies`
  - Merge into: `main`
  - Pin @&#8203;beepbeepgo/eslint-config-beepbeepgo-common to `1.0.4`
  - Pin [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) to `7.31.10`

</details>

---

❓ Got questions? Check out Renovate's [Docs](https://docs.renovatebot.com/), particularly the Getting Started section.
If you need any further assistance then you can also [request help here](https://github.com/renovatebot/renovate/discussions).

---

This MR has been generated by [Renovate Bot](https://github.com/renovatebot/renovate).

**Comment by iamjohnnym** [comment]
In GitLab by @group_13568306_bot1 on Oct 19, 2022, 19:21

:tada: This MR is included in version 1.0.1 :tada:

The release is available on:
- [GitLab release](https://gitlab.com/beepbeepgo/public/libraries/npm/nodejs-semantic-release-config/-/releases/1.0.1)
- `npm package (@latest dist-tag)`

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package: :rocket:
